styles/footer.css

```diff
@@ -0,0 +1,238 @@
+.footer {
+  display: flex;
+  flex-direction: column;
+  list-style: none;
+}
+
+@media screen and (min-width: 600px) {
+  .footer {
+    display: flex;
```

Comment:
o not duplicate styles inside media query

Reply:
done